src/orders/components/OrderFulfilledProductsCard/OrderFulfilledProductsCard.tsx

```diff
@@ -94,7 +94,7 @@ const OrderFulfilledProductsCard: React.FC<OrderFulfilledProductsCardProps> = pr
           <TableHeader />
           <TableBody>
             {renderCollection(getLines(), line => (
-              <TableLine line={line} />
+              <TableLine line={line} isFulfilled={true} />
```

Comment:
```suggestion
              <TableLine line={line} />
```

Reply:
isFulfilled is required prop for TableLine and doesn't have a default value. In OrderFulfilledProductsCard component we manage only fulfilled products, so the value is always `true` here.